weasel/tests/test_version_validation.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from weasel._util import validate_spacy_version
+
+
+@pytest.mark.parametrize(
+    "specifier,should_exit",
+    [
+        ("3.5.1", False),
+        (">=3.5.1", False),
+        (">=3.5.1", False),
+        (">3.0", False),
+        (">3.4", False),
+        ("!=3.5.1", True),
+        ("3.5.0", True),
+    ],
+)
+def test_spacy(specifier: str, should_exit: bool):
```

Comment:
I think we should add this test to spaCy instead. There, we can run it as part of the test suite and maybe even generate the options dynamically by fetching the current version of the repo, parsing it into `major.minor.patch` and create compatible & incompatible ranges.

Reply:
I don't know. This is extremely awkward, I agree – but it's still a Weasel functionality...

However, I will say the current test doesn't make much sense. I think we can monkeypatch `version` here, and do away with the `spacy` dependency. It's a bit flaky, but since it's a transient state until `spacy project` is treated as an alias, I think we're ok.

Reply:
I've monkeypatched `version`. I'm marking this conversation as resolved, but let me know if you'd rather we stopped testing for this altogether.